en/intro/conventions.rst

```diff
@@ -77,7 +77,7 @@ Database Conventions
 
 Table names corresponding to CakePHP models are plural and underscored. For
 example ``users``, ``article_categories``, and ``user_favorite_pages``
-respectively.
+respectively. Table name whose name contains multiple words should be singular_plural: article_categories and accordingly whole column. 
```

Comment:
>  and accordingly whole column.

I wanted to say whole third column of the table you have added should be updated to use singular _articles_ in `ArticleCategories/ArticleCategory/article_categories/...`

Reply:
I not speak english.... I'm doing like blind... Write down how the sentence should look, please.